README.md

```diff
@@ -1,357 +1,134 @@
-It can take one of the following options:
-
-- `built-in`: Use the compiled built-in contracts
-- `built-in-no-verify`: Use the compiled built-in contracts, but without signature verification
-- `local`: Load contracts from `ZKSYNC_HOME` or specify path using `--system-contracts-path`
+</details>
 
-**Example:**
+<details>
+<summary><strong>Script (installs forge-zksync, cast-zksync & anvil‑zksync)</strong></summary>
 
 ```bash
-anvil-zksync --dev-system-contracts local --system-contracts-path ./system-contracts --protocol-version 28
+curl -L https://raw.githubusercontent.com/matter-labs/foundry-zksync/main/install-foundry-zksync | bash
 ```
 
-## 📃 Logging
+</details>
 
-The node may be started in either of `debug`, `info`, `warn` or `error` logging levels via the `--log` option:
-```bash
-anvil-zksync --log=error run
-```
+<details>
+<summary><strong>Pre‑built binaries</strong></summary>
 
-Additionally, the file path can be provided via the `--log-file-path` option (defaults to `./anvil-zksync.log`):
 ```bash
-anvil-zksync --log=error --log-file-path=run.log run
+# download and unpack the latest release
+tar -xzf anvil-zksync_x.y.z_<platform>.tar.gz -C /usr/local/bin
+chmod +x /usr/local/bin/anvil-zksync
 ```
 
-The logging can be configured during runtime via the [`config_setLogLevel`](./SUPPORTED_APIS.md#config_setloglevel) and [`config_setLogging`](./SUPPORTED_APIS.md#config_setlogging) methods.
-
-## 📊 Telemetry
+</details>
 
-Anonymous usage data is collected **only if you opt in** when first prompted. You can opt out at any time by editing or removing the telemetry configuration file:
-
-- macOS (Darwin):
-
-  ```bash
-  $HOME/Library/Application Support/com.matter-labs.zksync-tooling/telemetry.json
-  ```
-
-- Linux:
-
-  ```bash
-  $XDG_CONFIG_HOME/zksync-tooling/telemetry.json
-  or
-  $HOME/.config/zksync-tooling/telemetry.json
-  ```
-
-**What we collect**:
-
-- Basic usage statistics  
-- Error reports  
-- Platform information  
-
-**What we do NOT collect**:
-
-- Personal information  
-- Sensitive configuration  
-- Private keys or addresses
-
-## 📃 Caching
-
-The node will cache certain network request by default to disk in the `.cache` directory. Alternatively the caching can be disabled or set to in-memory only
-via the `--cache=none|memory|disk` parameter.
+<details>
+<summary><strong>Build from source</strong></summary>
 
 ```bash
-anvil-zksync --cache=none run
+# clone and build from source
+git clone git@github.com:matter-labs/anvil-zksync.git
+cd anvil-zksync
+cargo build --release
 ```
 
-```bash
-anvil-zksync --cache=memory run
-```
+</details>
 
-Additionally when using `--cache=disk`, the cache directory may be specified via `--cache-dir` and the cache may
-be reset on startup via `--reset-cache` parameters.
-```bash
-anvil-zksync --cache=disk --cache-dir=/tmp/foo --reset-cache run
-```
+## ⚡️ Quick‑start
 
-## 🌐 Network Details
+1. **Start your local zkSync node**
```

Comment:
Numbering makes it seems like these are supposed to be executed sequentially as opposed to being independent options to run anvil-zksync. Maybe just small subsections instead? But no strong opinion